baski/server/async_server.py

```diff
@@ -99,6 +99,24 @@ def config(self):
         logging.info('Config file %s loaded', self.args['config'])
         return cfg
 
+    def update_config(self) -> None:
+        self.config.load_yml(self.args['config'])
+        self.config.load_db(firestore.Client())
+
+    def get_all_config_values(self) -> list:
+        return [cfg for cfg in self.config.values()]
+
+    async def check_update_config(self):
+        current_config = self.get_all_config_values()
+        while True:
+            self.update_config()
+            new_config = self.get_all_config_values()
+            if new_config != current_config:
+                logging.info('Config file update detected. Stop and close all tasks!')
+                self.stop()
```

Comment:
There is another loggin in self.stop() function. We will get confused if we get both lines. Please move out logging from the stop function to the callers